.golangci.yml

```diff
@@ -0,0 +1,35 @@
+version: "2"
+linters:
+  disable:
+    - errcheck      # TODO(rsned): Enable once outstanding lint bugs are fixed.
+    - mnd
```

Comment:
Is there any reason to disable the linters that aren't firing?  AFAICT, the only ones that are firing are:
```
- errcheck
- staticcheck
- unused
```
mnd seems useful.  We don't want testpackage but the default configuration isn't actually firing.

Reply:
Actually all of the exclusions ones trigger if enabled.  Added comments to each one on what and why.

Reply:
OK, there must be something wrong with my test setup then.

Reply:
For this initial push we are just enabling the formatting checks and checks that pass without changes.